scisim/api.py

```diff
@@ -0,0 +1,115 @@
+from scisim import app, db, clean_text
+from flask import request
+from models import *
+from scisim.helpers import *
+from sqlalchemy import and_
+from datetime import datetime
+from json import dumps
+
+@app.route('/api/login')
+def login():
```

Comment:
It might be a good call to prefix all the API view functions with "api_" or something like that. What do you think?